src/Type/Components/MapTypes.ts

```diff
@@ -0,0 +1,26 @@
+export interface MarkerData {
+    position: [number, number]
```

Comment:
this

Reply:
in map actually they are expecting value like this -> [number , number]